### PR TITLE
Add Supported Docker Containers for Linux x86

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

**Review by gdams — CHANGES_REQUESTED** [review]
One change

File: config/hotspot.config [hunk 1]
```diff
@@ -113,6 +113,18 @@ Architectures: windows-nano
 OS_Family: windows
 Directory: 11/jdk/windows/nanoserver-1809
 
+Build: releases
+Type: full
+Architectures: x86_64
```
[the remark this answers is not in the document]
Comment (karianna) [reply config/hotspot.config]: Updated - PTAL

[the remark this answers is not in the document]
Comment (karianna) [reply config/hotspot.config]: Actually, I don't think making that change is correct as we don't have the aarch64 or ppc64-le builds ready yet (for example)

[the remark this answers is not in the document]
Comment (gdams) [reply config/hotspot.config]: that's fine, the updater will automatically look for those platforms and skip them if they are unavilable.

**Comment by karianna** [comment]
The dockerhub_docs script isn't producing the correct output after this PR (it still only lists amd64

**Pull request merged**